conans/test/unittests/model/editable_cpp_info/parse_test.py

```diff
@@ -5,96 +5,41 @@
-        self.assertFalse(len(data['includedirs']))
-        self.assertFalse(len(data['libdirs']))
-        self.assertFalse(len(data['resdirs']))
-        self.assertFalse(len(data['bindirs']))
-
-    def test_read_empty_namespace(self):
-        save(self.layout_filepath, base_content.format(namespace="", path_prefix=""))
-        editable_cpp_info = EditableCppInfo.load(self.layout_filepath, require_namespace=True)
-        self.assertTrue(editable_cpp_info._uses_namespace)
-        self.assertFalse(len(editable_cpp_info._data.keys()))
-
-    def test_no_namespace(self):
-        save(self.layout_filepath, self.content)
-        editable_cpp_info = EditableCppInfo.load(self.layout_filepath, require_namespace=False)
-        self.assertFalse(editable_cpp_info._uses_namespace)
-
-        data = editable_cpp_info._data
-        self.assertListEqual(sorted(data.keys()), sorted(EditableCppInfo.cpp_info_dirs))
-        self.assertListEqual(list(data['includedirs']), ['dirs/includedirs', ])
-        self.assertListEqual(list(data['libdirs']), ['dirs/libdirs', ])
-        self.assertListEqual(list(data['resdirs']), ['dirs/resdirs', ])
-        self.assertListEqual(list(data['bindirs']), [])
-
-    def test_namespace(self):
-        save(self.layout_filepath, self.content)
-        editable_cpp_info = EditableCppInfo.load(self.layout_filepath, require_namespace=True)
-        self.assertTrue(editable_cpp_info._uses_namespace)
-
-        self.assertListEqual(sorted(editable_cpp_info._data.keys()), sorted(['*', 'libA', ]))
-
-        # Check '*'
-        data = editable_cpp_info._data['*']
-        self.assertListEqual(sorted(data.keys()),
-                             sorted(EditableCppInfo.cpp_info_dirs))
-        self.assertListEqual(list(data['includedirs']), ['all/dirs/includedirs', ])
-        self.assertListEqual(list(data['libdirs']), ['all/dirs/libdirs', ])
-        self.assertListEqual(list(data['resdirs']), ['all/dirs/resdirs', ])
-        self.assertListEqual(list(data['bindirs']), [])
-
-        # Check 'libA'
-        data = editable_cpp_info._data['libA']
-        self.assertListEqual(sorted(data.keys()),
-                             sorted(EditableCppInfo.cpp_info_dirs))
-        self.assertListEqual(list(data['includedirs']), ['libA/dirs/includedirs', ])
-        self.assertListEqual(list(data['libdirs']), ['libA/dirs/libdirs', ])
-        self.assertListEqual(list(data['resdirs']), ['libA/dirs/resdirs', ])
-        self.assertListEqual(list(data['bindirs']), [])
-
     def tearDown(self):
         shutil.rmtree(self.test_folder)
+
+    def field_error_test(self):
+        content = textwrap.dedent("""
+                            [includedrs]
+                            something
+                            """)
+        save(self.layout_filepath, content)
+        with self.assertRaisesRegexp(ConanException, "Wrong cpp_info field: includedrs"):
```

Comment:
Is this exception catch elsewhere and more context added to it? Things like: this is about a layout for editable files, this file is located in this path,...

Reply:
I am not sure I understand your comment. If the file that defines the layout contains unexpected syntax, shouldn't it raise?